### PR TITLE
completed assessment

### PR DESCRIPTION
**Pull request by JakeRoyRandall**

### CODE REVIEW

File: react-commenting.js [hunk 1]
```diff
@@ -111,14 +126,14 @@ class Board extends Component{
...
   }
 
   render(){
     return(
       <div id="square" onClick={ this.handleSquareClick }>
-        {/* 13) Here: */}
+        {/* 13) Here: we are showing on the screen the props of "value" that was passed in*/}
         { this.props.value }
       </div>
     )
```
Comment (sjproctor) [inline react-commenting.js:139]: Nice!

File: react.js [hunk 2]
```diff
@@ -5,61 +5,84 @@
...
-for(let i=0; i<names.length; i++){
-  console.log(`${names[i]} is ${names[i].length} characters long.`)
-}
+// for(let i=0; i<names.length; i++){
+//   console.log(`${names[i]} is ${names[i].length} characters long.`)
+// }
 
+names.map((v,i) => {
+  console.log(`${names[i]} is ${names[i].length} characters long.`);
+})
```
Comment (sjproctor) [inline react.js:38]: ${names[i]} can also be written as ${v} - it does the same thing but if you have value you might as well use it.

File: react.js [hunk 3]
```diff
@@ -5,61 +5,84 @@
...
 
 // 4. Write a React method that would add one and update the state of the count each time the method is called.
 
+
 this.state = {
   count: 0
 }
 
-
+this.setState({count: this.state.count++})
```
Comment (sjproctor) [inline react.js:58]: Functionality looks great. It just needs a method that is called to update the state.

File: react.md [hunk 4]
```diff
@@ -5,39 +5,75 @@ Answer the following questions. First, without external resources. Challenge you
...
+    Source: https://flaviocopes.com/react-state-vs-props/
+
+    In a React component, props are variables passed to it by its parent component. State on the other hand is still variables, but directly initialized and managed by the component.
+
 
 
 5. How would you explain state to a friend who doesn't know code?
 
-  Your answer:
+    Your answer: Imagine you have a dresser. Typically the dresser is closed. Sometimes it can have any number of drawers open or closed. We want to keep track of these different possibilities in our code so that we could access the different clothes inside when we need to. So we'd use the state of our dresser (i.e. drawer1: closed, drawer2: open, etc.) to give us that functionality in React.
```
Comment (sjproctor) [inline react.md:79]: Love it.

**Review by sjproctor — APPROVED** [review]
Great!

File: react.js (hunk 2 above)
Comment (JakeRoyRandall) [reply react.js:38]: Duh, I'm so silly. I just lazily copied and pasted everything after the for loop bracket. Thank you.

**Pull request merged**